projects/kubernetes/kubernetes/1-18/patches/0010-EKS-PATCH-refine-aws-loadbalancer-worker-node-SG-rul.patch

```diff
@@ -117,65 +117,11 @@ diff --git a/staging/src/k8s.io/legacy-cloud-providers/aws/aws_test.go b/staging
 index b4549a80527..720de76d58e 100644
 --- a/staging/src/k8s.io/legacy-cloud-providers/aws/aws_test.go
 +++ b/staging/src/k8s.io/legacy-cloud-providers/aws/aws_test.go
-@@ -2024,3 +2024,116 @@ func TestConstructStsEndpoint(t *testing.T) {
+@@ -2024,3 +2024,62 @@ func TestConstructStsEndpoint(t *testing.T) {
  		require.NoError(t, err)
  	})
  }
 +
```

Comment:
These were some fargate tests that slipped in unnoticed because we were never building the test files.

Reply:
Good catch 💯